src/integrations/uport/index.js

```diff
@@ -70,7 +70,7 @@ class Uport extends InjectedWeb3 {
   }
 
   getNetworkId() {
-    return hexWithoutPrefix(this.web3.currentProvider.networkId)
+    return +hexWithoutPrefix(this.web3.currentProvider.networkId)
```

Comment:
Do you need it as a number? I think this can be done more nicely. Why do you need it to be a number and where?

Reply:
`IsConnectedToCorrectNetwork` selector compares it against a number @andre-meyer